.travis.yml

```diff
@@ -10,7 +10,7 @@ cache:
 # cf. https://stackoverflow.com/a/52387639
 install:
 - travis_retry gem install s3_website -v 3.4.0
-- travis_retry pip install awscli --upgrade --user
+#- travis_retry pip install awscli --upgrade --user
```

Comment:
I seem to recall that @dougmartin figured out that if you install a newer version of python (3.6? 3.7?) before the `pip install awscli` line that everything works and you can keep the invalidation. The down side of removing the invalidation is that you may have to wait a bit from the end of the build before it can be tested. (It's supposed to be ~10 minutes but we've seen it appear to take longer.)

Reply:
I found this in the [Teacher Edition Tips Plugin](https://github.com/concord-consortium/teacher-edition-tips-plugin) repo by using GitHub search:
```
# the next command is to fix "SSLContext object is not available" error in awscli install
- travis_retry pyenv global 3.6
```

Reply:
Yup, that solution works.  I'm going to close this PR and open a new one.